src/components/toggle/toggle.test.tsx

```diff
@@ -12,4 +12,39 @@ describe('EuiToggle', () => {
 
     expect(component).toMatchSnapshot();
   });
+
+  describe('props', () => {
+    test('is disabled', () => {
+      const component = render(
+        <EuiToggle label="Is toggle on?" isDisabled {...requiredProps} />
```

Comment:
Remove all `requiredProps` from tests except the first one

```suggestion
        <EuiToggle label="Is toggle on?" isDisabled />
```